src/app/login/page.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+import Link from "next/link";
+import React, { useEffect } from "react";
+import axios from "axios";
+import { useRouter } from "next/navigation";
+import toast from "react-hot-toast";
+import { loginSchema } from "@/helpers/validationSchema";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+type FormValues = {
+  email: string;
+  password: string;
+};
+
+const Login: React.FC = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormValues>({
+    resolver: yupResolver(loginSchema),
+  });
+  const router = useRouter();
+
+  const onSubmit = async (data: FormValues) => {
+    try {
+      const response = await axios.post("/api/users/login", data);
+      console.log("login success", response.data);
+      toast.success("Login Success");
+      localStorage.setItem("role", response.data.role);
+
+      if (response.data.role === "admin") {
+        router.push("/");
+      } else if (response.data.role === "restaurantOwner") {
+        router.push("/");
+      } else {
+        router.push("/");
+      }
```

Comment:
- Why if/else condition for same output?

Reply:
Remove if/else now , Earlier I move users to different routes based on role.